src/Features/Core/Portable/CodeLens/CodeLensReferencesService.cs

```diff
@@ -177,24 +179,20 @@ private static SyntaxNode GetEnclosingCodeElementNode(Document document, SyntaxT
                         .Select(location => GetDescriptorOfEnclosingSymbolAsync(solution, location, cancellationToken))
                         .ToArray();
 
-                    await Task.WhenAll(referenceTasks).ConfigureAwait(false);
```

Comment:
we don't need this pattern. if cancellation throws inside of WhenAll, it will throw right cancellation exception.